src/main.cpp

```diff
@@ -58,10 +61,10 @@ int main( int argc, char* argv[] ) {
     Geometry* cracked = Geometry::Crack( geometry, crack_pattern );
 
     Scene* scene = new Scene( camera );
+    scene->AddGeometry( geometry );
+    scene->AddGeometry( sketch );
     scene->AddGeometry( cracked );
```

Comment:
You can comment one of these out so we don't have two geometries on display for now